specs/slider/Spec.md

```diff
@@ -32,167 +43,177 @@ Sliders are great for adjusting settings that reflect intensity levels (volume,
 
-Accessibility for slider is mostly covered with keyboard behavior (according to ARIA docs).
+1. drag handle over slider bar 
+2. click on the slider bar to select the value from the range.
+   NOTE: user should be able to grab the handle from any point in the slider area
 
-##### Focus
+Changing the value is performed **from current value** to the next expected value. 
+E.g. if min=0, max=10, step=2, value=3.5, then UP arrow key will give us 4 and Down arrow key will give us 2
+Value can not exceed the min/max limits. If value is > or < than min/max it is automatically set to corresponding min/max.
 
-Slider should have a focus state. Focus state looks like a square border around handle. We place focus around handle because in range slider there is no other options and we need to keep consistency in all components. 
-![sliderFocus](./assets/sliderFocus.png)
+If slider has a 'step' prop, handle should move across the slider bar only according to the step.  
 
-reference links: 
-https://www.paciellogroup.com/blog/2008/05/aria-slider-part-1/
-https://www.paciellogroup.com/blog/2008/06/aria-slider-part-2/
+#### Validation
 
+| Case                          | Handling          |
+| ----------------------------- | ----------------- |
+| `value` is less then `min`    | set to `min`      |
+| `value` is greater than `max` | set to `max`      |
+| `value` is empty              | set to `min` OR 0 |
 
 
-## Examples
 
-#### Simple
+#### Edge case handling
 
-The `defaultValue` property sets the initial position of the slider. 
+| Case                                     | Handling                                 | Example                                  |
+| ---------------------------------------- | ---------------------------------------- | ---------------------------------------- |
+| value is out of `step`                   | keep current value & display handle at the position that represents this value | `min=0` `max=10` `step=2` `value=3.5`. Handle is displayed at the position that represents 3.5. When user changes value (keyboard / mouse) we move it to the next expected, which is 4 or 2 |
+| `step` is not set & `displayMarks` is set to true | show marks at the beginning and at the end of the slider |                                          |
 
-![simpleExample](./assets/simpleExample.png)
 
 
+### Input Methods 
 
-#### Stepped
+#### Keyboard
 
-By default, the slider is continuous. The step property causes the slider to move in discrete increments.
+| Keys                                 | Action                      |
+| ------------------------------------ | --------------------------- |
+| up / right arrow key                 | increase value              |
+| left / down arrow key                | decrease value              |
+| home ( fn/ctrl + left arrow key)     | set min value               |
+| end ( fn/ctrl + left arrow key)      | set min value               |
+| page up (fn/ctrl + up arrow key)     | increase value by X         |
```

Comment:
Well, but it should;) 
Look here - https://www.w3.org/TR/wai-aria-practices/examples/slider/slider-2.html
http://joxi.ru/Dr8jepkCkKyOX2